setup

```diff
@@ -742,10 +742,15 @@ def configure_default_compilers(args):
             args.fc  and os.path.basename(args.fc).lower().startswith('openmpi'):
             args.mpi = 'on'
 
-    if not args.cc:
-        args.cc = 'gcc'
-    if not args.cxx:
-        args.cxx = 'g++'
+    if not args.cc and os.environ['CC']:
```

Comment:
Ah, none of my tests caught that. I'll get a fix committed very shortly. I had CC/CXX/FC defined.